Do not look for libintl headers outside of the prefix.
Do record the value of LDFLAGS at configure time (https://github.com/eribertomota/axel/issues/1)
--- configure.orig	2015-11-01 16:48:49.000000000 -0600
+++ configure	2015-11-13 10:11:08.000000000 -0600
@@ -114,13 +114,7 @@
 if [ "$i18n" = "1" ]; then
 	echo 'I18N=1' >> Makefile.settings
 	echo '#define I18N' >> config.h
-	if [ -f /usr/local/include/libintl.h ]; then
-		AXEL_CFLAGS="${AXEL_CFLAGS} -I/usr/local/include"
-		AXEL_LFLAGS="${AXEL_LFLAGS} -L/usr/local/lib"
-	elif [ -f /opt/csw/include/libintl.h ]; then
-		AXEL_CFLAGS="${AXEL_CFLAGS} -I/opt/csw/include"
-		AXEL_LFLAGS="${AXEL_LFLAGS} -L/opt/csw/lib"
-	elif [ -f "${prefix}/include/libintl.h" ]; then
+	if [ -f "${prefix}/include/libintl.h" ]; then
 		AXEL_CFLAGS="${AXEL_CFLAGS} -I${prefix}/include"
 		AXEL_LFLAGS="${AXEL_LFLAGS} -L${prefix}/lib"
 	fi
@@ -236,6 +230,7 @@
 
 echo "CFLAGS+=${AXEL_CFLAGS}" >> Makefile.settings
 echo "LFLAGS+=${AXEL_LFLAGS}" >> Makefile.settings
+echo "LDFLAGS+=${LDFLAGS}" >> Makefile.settings
 
 echo 'Configuration done:'
 if [ "$i18n" = "1" ]; then
